packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -468,6 +468,49 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           });
           return state.bookDataKit.recorder.write(bookData);
         },
+        observeQuoteNotifier() {
+          const { state, facets } = this;
+          const { collateralBrand, bidBrand, priceAuthority } = state;
+          assertAllDefined({ collateralBrand, bidBrand, priceAuthority });
```

Comment:
this is checked in `init` before they are put into `state`, so it's redundant on pulling out again. 
```suggestion
```